featuros-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,23 @@
+project(':featuros-gradle-plugin') {
+    apply plugin: 'groovy'
+    apply plugin: 'maven-publish'
+
+    dependencies {
+        compile gradleApi()
+        compile localGroovy()
+
+        testCompile 'org.spockframework:spock-core:1.0-groovy-2.3'
```

Comment:
I guess there is no point to try to override it then as runtime environment will be 2.3.3 anyway.